frontend/markdown-pdf/build.js

```diff
@@ -6,6 +6,10 @@ const markdownit = require('markdown-it');
 const path = require('path');
 const puppeteer = require('puppeteer');
 const vuepressUtil = require('vuepress/lib/util');
+const hummus = require('hummus');
+const memoryStreams = require('memory-streams');
+var QRCode = require('qrcode');
+var moment = require('moment');
```

Comment:
Je definieert sommige variablen als constant (`const`), anderen met het oudere en minder strikte `var`. Kan je even checken of je voor deze cases `const` of `let` kunt toepassen?

Reply:
Whoops, old habits die hard